pkgs/applications/networking/seafile-client/default.nix

```diff
@@ -1,23 +1,42 @@
-{ mkDerivation, lib, fetchFromGitHub, pkg-config, cmake, qtbase, qttools
-, seafile-shared, jansson, libsearpc
-, withShibboleth ? true, qtwebengine }:
+{ mkDerivation
+, lib
+, fetchFromGitHub
+, pkg-config
+, cmake
+, qtbase
+, qttools
+, libuuid
+, seafile-shared
+, jansson
+, libsearpc
+, withShibboleth ? true
+, qtwebengine
+}:
 
 mkDerivation rec {
   pname = "seafile-client";
-  version = "8.0.7";
+  version = "9.0.1";
 
   src = fetchFromGitHub {
     owner = "haiwen";
     repo = "seafile-client";
-    # Tag v8.0.7 used to point to this commit, but was re-tagged later to a state
-    # that fails to link properly
-    rev = "8b96abf64e6aa19e3beeff6b322bcd6d28120bdd";
-    sha256 = "00wfr7dvbyl7pg1xgssgz8a94c7c4n5r9266lhy9qcbz456hdcgj";
+    rev = "v${version}";
+    sha256 = "sha256-4PIkjjqyE8WmYJ2z2N4oruv7AlX+zXo8lB9Rmc8sn4Y=";
   };
 
-  nativeBuildInputs = [ pkg-config cmake ];
-  buildInputs = [ qtbase qttools seafile-shared jansson libsearpc ]
-    ++ lib.optional withShibboleth qtwebengine;
+  nativeBuildInputs = [
+    libuuid
+    pkg-config
+    cmake
+  ];
+
+  buildInputs = [
+    qtbase
+    qttools
```

Comment:
I guess `qttools` should be in `nativeBuildInputs`?

Reply:
I'm not sure.
Should I include the optional `qtwebengine` in the `nativeBuildInputs` as well?

Reply:
No, qtwebengine is buildInputs.

Reply:
Ok, moved back to `buildInputs`.